docs/kubeadm.md

```diff
@@ -0,0 +1,243 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+This quickstart shows you how to easily install a secure Kubernetes cluster on machines running Ubuntu 16.04 or CentOS 7, using a tool called `kubeadm` which is part of Kubernetes.
```

Comment:
Run-on sentence.

"This quickstart shows you how to install a secure Kubernetes cluster on machines running Ubuntu 16.04 or CentOS 7. The installation uses a tool called `kubeadm`, which is part of Kubernetes 1.4."